README.md

```diff
@@ -316,6 +316,19 @@ $ python -m pip install "./python[spark]"
 
 ### Run Tests
 
+Some FlinkProcessor's python tests require a Hadoop environment to function
```

Comment:
Given that running Flink unit tests (which presumably should include Parquet-related unit tests) do not require developers to manually download and setup Hadoop environment, can you explain why running FeatHub unit tests requires this setup?

Reply:
In Flink unit tests, parquet-related tests would be skipped if HADOOP_CLASSPATH is not configured[1]. I think it would be better to enforce the configuration of hadoop environment rather than skippting these tests in Feathub.

[1] https://github.com/apache/flink/blob/release-1.16/flink-python/pyflink/datastream/formats/tests/test_parquet.py#L55